test/subset-band-name/utilities.py

```diff
@@ -0,0 +1,118 @@
+""""
+Common utility functions used by the subset-band-name regression tests.
+"""
+
+import os
+from pyhdf.SD import SD, SDC
+import numpy
+
+from harmony import Client, Request
+from harmony.harmony import ProcessingFailedException
+
+
+def submit_and_download(
```

Comment:
Could the [shared utility function](https://github.com/nasa/harmony-regression-tests/blob/main/test/shared_utils/utilities.py#L33) for `submit_and_download` be used instead of having a near-duplicate here? I see there is a minor difference (the use of file indicator), but looking at the `/jobs` endpoint for the request I ran, I think only one file is listed in the response, so I _think_ you can just use that other version of the function to download that one file.

Similarly, those common utilities have `print_error` and `print_success`.

@flamingbear added some [helpful instructions](https://github.com/nasa/harmony-regression-tests/blob/main/test/shared_utils/README.md) for how to include the shared utility functions in your Docker image.

Reply:
Actually the reference files are HDF4, and I can't get them to open with h5python or netcdf4 which is not fully unexpected.  So I won't be able to use the comparison for the shared library. It would have the print_error and print_success, but maybe another time?